tests/static/js/jqunit/jqUnit-binder-commonEventBindingsOverride.js

```diff
@@ -0,0 +1,177 @@
+                        }
+                    ]
+                },
+                {
+                    name: "Test markup event binding for both `click` and `keydown` on an input button",
+                    type: "test",
+                    expect: 2,
+                    sequence: [
+                        {
+                            func: "gpii.tests.binder.clickSelector",
+                            args: ["#input-multiple-events-button"]
+                        },
+                        {
+                            func: "gpii.tests.binder.keydownSelector",
+                            args: ["#input-multiple-events-button"]
+                        }
+                    ]
+                },
+                {
+                    name: "Test markup event binding for both `click` and `keydown` on an input button, each using a different handler",
+                    type: "test",
+                    expect: 2,
+                    sequence: [
+                        {
+                            func: "gpii.tests.binder.clickSelector",
+                            args: ["#input-multiple-handlers-events-button"]
+                        },
+                        {
+                            func: "gpii.tests.binder.keydownSelector",
+                            args: ["#input-multiple-handlers-events-button"]
+                        }
+                    ]
+                }
+            ]
+        }]
+    });
+
+    //
+    // Test for overriding binder markup events
+    //
+    fluid.defaults("gpii.tests.binder.commonEventBindingsOverride", {
+        gradeNames: ["gpii.tests.binder.commonEventBindings"],
+        markupEventBindings: {
+            // In the original grade this only had 1 method, overriding with 2
+            multipleEventInputButton: {
+                method: ["click", "keydown"],
+                args: ["{that}.handleInputClick"]
+            },
+            // This is a new set of events that was not in the original grade
+            multipleEventHandlerInputButton: [
+                {
+                    method: "click",
+                    args: ["{that}.handleInputClick"]
+                },
+                {
+                    method: "keydown",
+                    args: ["{that}.handleInputKeydown"]
+                }
+            ]
+        }
+    });
+
+    fluid.defaults("gpii.tests.binder.commonEventBindingsOverride.environment", {
+        gradeNames:       ["gpii.tests.binder.environment"],
+        markupFixture:    ".viewport-common-event-bindings",
+        binderGradeNames: ["gpii.tests.binder.commonEventBindingsOverride"],
+        moduleName:       "Testing common event bindings",
+        components: {
+            commonEventBindingsOverrideTests: {
+                type: "gpii.tests.binder.commonEventBindingsOverride.caseHolder"
+            }
+        }
+    });
+
+    gpii.tests.binder.commonEventBindingsOverride.environment();
```

Comment:
We should either consistently use `fluid.test.runTests` or consistently not use it.

Reply:
+1 since without runTests there is no teardown